db/schema.rb

```diff
@@ -10,11 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2023_01_27_123745) do
+ActiveRecord::Schema[7.0].define(version: 2023_02_14_083343) do
   create_sequence "qualifications_public_id_seq", start: 120000
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "pgcrypto"
+  enable_extension "plpgsql"
```

Comment:
Should we add this back? I am not sure but someone was asking in the apply tech about this and removed this line.

Reply:
Ah we likely don't need this and Rails has just stuck it in due to a setting on my local Postgres; I'll remove it.